.changeset/mighty-snakes-thank.md

```diff
@@ -0,0 +1,7 @@
+---
+'@cypress-design/react-icon': minor
+'@cypress-design/vue-icon': minor
+'@cypress-design/icon-registry': minor
```

Comment:
these are typically patch releases

Reply:
I was referencing https://github.com/cypress-io/cypress-design?tab=readme-ov-file#adding-a-new-icon - imo minor makes sense for net new (non-fix) additions?

Reply:
I tend to agree with minor as well, just the pattern I've seen in this repo for new icons